bin/cli.js

```diff
@@ -46,8 +48,8 @@ var defaultYargsInstance = yargs
   ].join('\n')))
```

Comment:
This is a prompt for opinion, it should not change this increment.

Currently the default instance is coupled to angularity. I would like to see it (eventually) as its own plugin. We could use a special name, such as `$` to indicate that it is the entry point.

There is a (minor?) trade off between speed of top level functions (i.e. `angularity -v`, `angularity -h`) but that is one that I would personally make.